src/main/scala/it/pagopa/pdnd/interop/uservice/partymanagement/common/system/ApplicationConfiguration.scala

```diff
@@ -15,7 +15,7 @@ object ApplicationConfiguration {
     config.getString("uservice-party-management.storage.type")
   }
 
-  def tokenValidity: Long = {
-    config.getLong("uservice-party-management.token-validity")
+  def tokenValidityHours: Long = {
```

Comment:
saying hours are we assuming that the env var must define hours, right? What if applications need a less than one hour validity though?

Sorry for the silly question, maybe I am missing some specs 😊

Reply:
At this moment, there are no specs about that.
I would extract this value from the code.